### PR TITLE
Engineering Station POI

### PR DESCRIPTION
**Pull request by Belsima**
i await someone to tell me how dumb i be

### CODE REVIEW

**Comment by Belsima** [comment]
good i got what i needed this was me testing so i could be directed to the info

**Comment by Anewbe** [comment]
In the future, PM the details of the PoI to a Dev, rather than posting them here.
Or rather, do that now, too, but in the future just start with that.

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -62,6 +62,16 @@
 	toolspeed = 0.25
 	var/obj/item/weapon/screwdriver/power/counterpart = null
 
+/obj/item/weapon/wrench/rusted
+	desc = "This wrench seems to have seen a few many days."
```
Comment (Atermonera) [inline code/game/objects/items/weapons/tools.dm]: `seen a few many`

File: maps/submaps/surface_submaps/wilderness/wilderness.dm [hunk 2]
```diff
@@ -192,4 +193,10 @@
 	name = "Musk"
 	desc = "0 to 60 in 1.9 seconds."
 	mappath = 'maps/submaps/surface_submaps/wilderness/Musk.dmm'
-	cost = 10
\ No newline at end of file
+	cost = 10
+
+/datum/map_template/surface/wilderness/normal/engi-station
```
Comment (Atermonera) [inline maps/submaps/surface_submaps/wilderness/wilderness.dm]: Travis might be angry about this? I've never seen a - in a type path.
Also an issue for the area name

Comment (Belsima) [reply maps/submaps/surface_submaps/wilderness/wilderness.dm]: Ahh... I'll change that, thanks.

File: code/game/objects/items/weapons/tools.dm (hunk 1 above)
Comment (Belsima) [reply code/game/objects/items/weapons/tools.dm]: ...huh. That's... not grammatically accurate. My life is a lie. I'd seen it used in person and in books, I feel so lost now. Will fix, thanks.

**Pull request closed without merging**